test/e2e/pkg/testnet.go

```diff
@@ -460,7 +459,7 @@ func (n Node) AddressRPC() string {
 
 // Client returns an RPC client for a node.
 func (n Node) Client() (*rpchttp.HTTP, error) {
-	return rpchttp.New(fmt.Sprintf("http://127.0.0.1:%v", n.ProxyPort), "/websocket")
+	return rpchttp.New(fmt.Sprintf("http://%s:%v", n.IP, n.ProxyPort), "/websocket")
```

Comment:
IIRC this change caused problems with the Docker-based backend previously on some people's machines. @jmalicevic do you perhaps remember whether you managed to resolve those problems?

Reply:
I'm seeing that as well. Taking a look.

Reply:
Fixed.